uploader/uploader.go

```diff
@@ -125,10 +150,11 @@ func (e *Uploader) Upload(ctx context.Context, local, remote string, ctypeFunc C
 	}
 
 	if fi.IsDir() {
+		log.WithField("Going to upload ", local).WithField("to remote", remote).Info(" - directory")
```

Comment:
Your with field here shoudl just have the ("local", local). It's not a "message", but a structured log entry

Reply:
Removed.